apps/prairielearn/src/components/QuestionsTable.html.ts

```diff
@@ -36,6 +39,10 @@ export function QuestionsTable({
   __csrf_token,
 }: {
   questions: QuestionsPageData[];
+  templateQuestions?: { qid: string; title: string }[];
+  /**
```

Comment:
Same comment as elsewhere, this comment should come before the line that it describes.